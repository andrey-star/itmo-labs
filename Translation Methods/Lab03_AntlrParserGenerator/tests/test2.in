def f(a):
	return __g__(a)

print(f(1))
print(__g__(2))
bb = int(input())

def __g__(b):
	return f(b)

ab_09 = f(1) + __g__(2) + f(__g__(bb))
print(ab_09)
