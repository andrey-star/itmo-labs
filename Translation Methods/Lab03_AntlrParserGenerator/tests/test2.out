int f(int a);
int __g__(int b);

int f(int a) {
	return __g__(a);
}

int __g__(int b) {
	return f(b);
}

int main() {
	printf("%d\n", f(1));
	printf("%d\n", __g__(2));
	int bb;
	scanf("%d", &bb);
	int ab_09 = f(1) + __g__(2) + f(__g__(bb));
	printf("%d\n", ab_09);
	return 0;
}
